repose-aggregator/artifacts/src/config/filters/log4j2.xml

```diff
@@ -26,9 +26,22 @@
         <Logger name="com.sun.jersey" level="off"/>
         <Logger name="net.sf.ehcache" level="error"/>
         <Logger name="org.apache" level="warn"/>
+        <Logger name="org.apache.http.wire" level="trace">
+            <Filters>
+                <ContextMapFilter onMatch="ACCEPT" onMismatch="NEUTRAL">
+                    <KeyValuePair key="x-trace-request" value="true"/>
+                </ContextMapFilter>
+                <ThresholdFilter level="WARN" onMatch="NEUTRAL" onMismatch="DENY"/>
+            </Filters>
+        </Logger>
         <Logger name="org.eclipse.jetty" level="off"/>
         <Logger name="org.springframework" level="warn"/>
-        <Logger name="intrafilter-logging" level="info"/>
+        <Logger name="intrafilter-logging" level="trace"/>
```

Comment:
I don't like this inconsistency

Reply:
Yeah, I went back on forth on whether or not to put the filter on this logger. At the moment, the filter isn't strictly necessary since we put the check in code, so I figured I would leave it off to keep the config simpler and cut out the miniscule overhead of having the filter run.

I wanted to include the filter to be explicit in the logging config about the preconditions for intrafilter logging, but then I was worried that if I did add the filter I would effectively create another precondition since there is nothing that would keep the filter check in-sync with the code check.

So ultimately, I opted to keep this logger simple, if inconsistent, and rely on documentation to tell users how to troubleshoot with intrafilter logging. If you would rather I add the filter, I'll go ahead and do that.

Reply:
You add it in